packages/asset-swapper/src/types.ts

```diff
@@ -192,6 +193,11 @@ export interface SwapQuoteOrdersBreakdown {
  */
 export interface SwapQuoteRequestOpts extends CalculateSwapQuoteOpts {
     gasPrice?: BigNumber;
+    apiKey?: string;
```

Comment:
Feels like apiKey should also be under `rfqt`, unless we're planning on using outside of the rfqt context.

Reply:
@fabioberger ?  will you be using `apiKey` here for meta transactions?  i coded it this way thinking that he would.

Reply:
Let's leave it top-level, we will be using it for other initiatives, including meta-txns.

Reply:
Fabio and I discussed this elsewhere.  He only needs to use an API key in the 0x API, not in asset-swapper, so I ended up putting `apiKey` under `rfqt` after all.